src/Lykke.Job.OrderBooksCacheProvider/Binders/JobModule.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Autofac;
+using Common.Log;
+using Lykke.Job.OrderBooksCacheProvider.Core;
+using Lykke.Job.OrderBooksCacheProvider.Core.Services;
+using Lykke.Job.OrderBooksCacheProvider.Services;
+using Lykke.RabbitMqBroker.Subscriber;
+using Lykke.SettingsReader;
+using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.Caching.Redis;
+using RestSharp;
+
+namespace Lykke.Job.OrderBooksCacheProvider.Binders
+{
+    public class JobModule : Module
+    {
+        private readonly OrderBooksCacheProviderSettings _settings;
+        private readonly ILog _log;
+
+        public JobModule(IReloadingManager<AppSettings> settings, ILog log)
+        {
+            _settings = settings.CurrentValue.OrderBooksCacheProvider;
+            _log = log;
+        }
+
+        protected override void Load(ContainerBuilder builder)
+        {
+            builder.RegisterType<HealthService>()
+                .As<IHealthService>()
+                .SingleInstance();
+
+            builder.RegisterType<StartupManager>()
+                .As<IStartupManager>();
+
+            builder.RegisterInstance(_log);
+            builder.RegisterInstance(_settings);
```

Comment:
This is a bad practice. It's better to provide parameters to components registration from settings.